Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -50,6 +50,56 @@ Building a reverse engineering environment for free is possible. However, there
 
+In iOS, all the application code, for both Swift and Objective-C, is compiled to machine code (e.g. ARM). Thus, to analyze iOS application a disassembler is needed. 
+
+If you want to disassemble an application's code from the Apple Appstore, remove the Fairplay DRM first. Section "[Acquiring the App Binary](0x06b-basic-security-testing.md#acquiring-the-app-binary "Acquiring the App Binary")" in the chapter "iOS Basic Security Testing" explains how. 
+
+In this section the term "app binary" refers to the Macho-O file in the application bundle which contains the compiled code, and should not be confused with the application bundle - the IPA file. See section "[Exploring the App Package](0x06b-basic-security-testing.md#exploring-the-app-package "Exploring the App Package")" section in chapter "Basic iOS Security Testing" for more details on the composition of IPA files. 
+
+##### Disassembling With Ghidra
+As mentioned in "[Tooling](#tooling "Tooling")" section, Ghidra is an open source software reverse engineering tool. Among multiple functionalities offered by Ghidra, it can also be used to analyze iOS application binaries. 
+
+To get started with Ghidra is easy. Start Ghidra using `ghidraRun` (\*nix) or `ghidraRun.bat` (Windows), depending on the platform you are on. Once Ghidra is fired up, create a new project by specifying the project directory. You will be greeted by a window as shown below:  
+
+![Ghidra New Project Window](Images/Chapters/0x06c/Ghidra_new_project.png)
+
+In your newly created project space from above, you can import the app binary into this project, by going to **File** -> **Import File** and choosing the desired file. 
+
+![Ghidra import file](Images/Chapters/0x06c/Ghidra_import_binary.png)
+
+If the correct file is specified, Ghidra will show meta-information about the binary before starting the analysis. 
+
+![Ghidra Mach-O file import](Images/Chapters/0x06c/Ghidra_macho_import.png)
+
+To get the disassembled code for the binary file choosen above, double click the imported file. Click **yes** and **analyze** for auto-analysis on the subsequent windows. Auto-analysis will take some time depending on the size of the binary, the progress can be tracked in the bottom right corner of the code browser window. Once auto-analysis is completed you can start exploring the binary.  
+
+![Ghidra code browser window](Images/Chapters/0x06c/Ghidra_main_window.png)
+
+In above annotated main window of Ghidra, disassembly window and symbol tree window are most important for moving around and exploring the binary. Decompiler window shows the decompiled version of the function selected for disassembly. It is important to note decompilation is not always accurate, but nevertheless very helpful in getting a quick understanding of the function being analyzed. It is always a good idea to keep cross-checking decompiled code against disassembled code for maximum accuracy.
```

Comment:
```suggestion
In the main window of Ghidra, the disassembly window and symbol tree window are the most important for moving around and exploring the binary. The Decompiler window shows the decompiled version of the selected function for disassembly. It is important to note that decompilation is not always accurate, but nevertheless can be very helpful in getting a quick understanding of the analyzed function being. It is always a good idea to keep cross-checking decompiled code against disassembled code for maximum accuracy.
```

Reply:
Can you please recheck this suggestion. I think you mistakenly changed the original sentence "....quick understanding of the function being analyzed" to 
   "....quick understanding of the analyzed function being".